core/src/main/scala/filodb.core/query/QueryConfig.scala

```diff
@@ -21,6 +21,8 @@ class QueryConfig(queryConfig: Config) {
     v
   }
   lazy val parser = queryConfig.as[String]("parser")
+  lazy val fastSubquery = if (queryConfig.hasPath("fastSubquery"))
+    queryConfig.as[Boolean]("fastSubquery") else true
```

Comment:
No need for fallback. Since filo-defaults.conf has it, it always will fallback to default.

Reply:
application_test.conf also needs to be changed for unit tests